### PR TITLE
PREAPPS-2821: Break account info query into multiple queries

### PR DESCRIPTION
**Pull request by lavegupta**

### CODE REVIEW

File: src/batch-client/index.ts [hunk 1]
```diff
@@ -175,26 +175,20 @@ export class ZimbraBatchClient {
...
-					license: {
-						status: res.license.status,
-						attr: mapValuesDeep(res.license.attr, coerceStringToBoolean)
-					}
-				})
-			};
-		});
+			namespace: Namespace.Account,
+			body: {
+				sections: 'mbox,attrs,zimlets'
```
Comment (itsmeprashant) [inline src/batch-client/index.ts]: FYI - This is to define the kind of data expected from response of GetInfoRequest.

File: src/batch-client/index.ts [hunk 2]
```diff
@@ -584,6 +578,11 @@ export class ZimbraBatchClient {
 			return c;
 		});
 
+	public getDataSources = () =>
```
Comment (itsmeprashant) [inline src/batch-client/index.ts]: FYI - To get IMAP and POP account data.

File: src/batch-client/index.ts [hunk 3]
```diff
@@ -175,26 +175,20 @@ export class ZimbraBatchClient {
...
-					}
-				})
-			};
-		});
+			namespace: Namespace.Account,
+			body: {
+				sections: 'mbox,attrs,zimlets'
+			}
+		}).then(res => ({
+			...res,
```
Comment (itsmeprashant) [inline src/batch-client/index.ts]: If we are changing return data here then we should update `type AccountInfo`, and entity if needed.

Comment (lavegupta) [reply src/batch-client/index.ts]: Not needed.

**Pull request merged**